src/primitives/tx_version_info.cpp

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2009-2010 Satoshi Nakamoto
+// Copyright (c) 2009-2014 The Bitcoin Core developers
```

Comment:
The extracted function is entirely our own code, so this header should actually be:
```suggestion
// Copyright (c) 2021 The Zcash developers
```